components/21-atoms/table/table.functions.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+var ResponsiveTable = function (element, options) { // eslint-disable-line no-unused-vars
+  /**
+   * The table wrapper.
+   */
+  var table = null;
+
+  /**
+   * The table caption.
+   */
+  var caption = null;
+
+  /**
+   * The options for the table.
+   */
+  var opts = {};
+
+  /**
+   * Determine if the table should be focusable.
+   */
+  var determineFocusable = function () {
+    var scrollableWidth = table.parentNode.querySelector('table').scrollWidth; // eslint-disable-line no-console
+    var containerWidth = table.parentNode.clientWidth; // eslint-disable-line no-console
+
+    // Check if element is scrollable.
+    if (scrollableWidth <= containerWidth) {
+      // If not remove the tab focus.
+      table.removeAttribute('tabindex');
+    }
+    else {
+      caption.innerText = caption.innerText + ' ' + opts.scrollableText;
```

Comment:
```suggestion
      caption.innerText += ' ' + options.scrollableText;
```

Also either:

* add a default value for `options.scrollableText`.
* add a check that throws an error when `options.scrollableText` is not provided.